src/components/Header/index.tsx

```diff
@@ -28,35 +56,25 @@ export const Header = () => {
     const dev = !!query.dev;
     const queryState = query.state;
 
+    const headerOptions = [...headerCategories];
+    dev && headerOptions.push(...devHeaderCategories);
+
     return (
         <div className={styles.header}>
-            {headerCategories.map(
-                ({ title, refURL, pathName, state }, index) => (
-                    <HeaderLink
-                        key={index}
-                        title={title}
-                        link={refURL}
-                        isActive={
-                            router.pathname === pathName &&
-                            (router.pathname === '/pull-requests'
-                                ? queryState === state
-                                : true)
-                        }
-                    />
-                )
-            )}
-
-            {dev &&
-                devHeaderCategories.map(
-                    ({ title, refURL, pathName }, index) => (
-                        <HeaderLink
-                            key={index}
-                            title={title}
-                            link={refURL}
-                            isActive={router.pathname === pathName}
-                        />
-                    )
-                )}
+            {headerOptions.map(({ title, refURL, pathName, state }, index) => (
```

Comment:
In this code, you're currently wrapping the array of `<HeaderLink>` components inside a `<div>` element in the `Header` component's return statement. However, you can use a React fragment instead to avoid adding an extra `<div>` to the DOM

Reply:
Do you mean the div on line 63? Need that for adding class.